src/main/webapp/vendor/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
```

Comment:
this should be in `vendor/awesomedashboard2/index.html` or something. Same with the LICENSE.